protocol/consensus.go

```diff
@@ -198,3 +198,7 @@ func (err Error) Error() string {
 	proto := ConsensusVersion(err)
 	return fmt.Sprintf("protocol not supported: %s", proto)
 }
+
+// SeedLookback is how many blocks back we use seeds from in sortition.
```

Comment:
why? this is a consensus parameter

Reply:
Well, it's essentially a constant. And changing this number with consensus upgrade will not magically work without extra upgrade code.

Reply:
And with the proposed change it will require even more code.  Moving a proto param into a const is no-go.
The ledger might use the latest persisted header to determine its lookback.